app/views/admin/merchants/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render partial: "admin/admin_nav" %>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<%# Refactor these br's later to format padding/argins better  %>
+<%= form_with model: Merchant.new, url: admin_merchants_path do |f| %>
```

Comment:
Hey, remember to call this in the controller and then just cal the variable in the view.

Reply:
Sorry, Merchant.new in the controller and @merchant in the view

Reply:
Thanks for the reminder! I'll update this in my current branch since this already got merged